machines/vic/certificate.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
Looks like this is one command line tool similar to OpenSSL, I think it will work well. 
If use this class, do we need to call the function through CLI?

Ian worked on this piece of code, I think the benefit of his work is that we can call that through go function call.
But I'll confirm with him in case there is other consideration for this design.